optuna/storages/_journal/redis.py

```diff
@@ -0,0 +1,66 @@
+import json
+import time
+from typing import Any
+from typing import Dict
+from typing import List
+
+from optuna._experimental import experimental_class
+from optuna._imports import try_import
+from optuna.storages._journal.base import BaseJournalLogStorage
+
+
+with try_import() as _imports:
+    import redis
+
+
+@experimental_class("3.1.0")
+class JournalRedisStorage(BaseJournalLogStorage):
+    """Redis storage class for Journal log backend.
+
+    Args:
+        url:
+            URL of the redis storage, password and db are optional. (ie: redis://localhost:6379)
+
+    """
+
+    def __init__(self, url: str) -> None:
```

Comment:
[nits] Might be good to provide a "prefix" argument for users who share the same redis DB among others.  Leaving a TODO comment is enough for now.

```suggestion
    def __init__(self, url: str, prefix: str) -> None:
```

Reply:
Thanks for the suggestion. I will add the argument, but define None by default.